### PR TITLE
Added meshery broader capability 

### LINKED ISSUE #5768: [Docs] Represent Meshery's broader management capabilities
**Current Behavior**
Meshery manages all kinds of cloud native infrastructure. Branding it as a service mesh manager only misrepresents the full-breadth of its capabilities.

**Desired Behavior**
Replace most, but not all, instances of "service mesh" with "cloud native". Examples: "service mesh manager", "service mesh management plane", "multi-mesh manager", "multi-mesh management plane"... these should be replaced with "cloud native manager".

---

#### [Contributor Guides](https://docs.meshery.io/project/contributing) and Resources
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by harkiratsm**
Signed-off-by: harkiratsm <multaniharry714@gmail.com>

**Description**

This PR fixes #5768 

**Notes for Reviewers**

Changed the typo for the following : 

- service mesh management -> cloud native manager
- service mesh management plane -> cloud native management plane
- multi mesh manager -> cloud native manager
- multi mesh management -> cloud native manager

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit f4a3692882c3dd7b1e4b4f499501031844eab91e at: https://62ce9474f4231d6c07f81b88--meshery-docs-preview.netlify.app

File: docs/_adapters/istio/istio.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ The ({{page.name}}) includes a handful of sample applications. Use Meshery to de
...
+As the service mesh management plane, Meshery enables the adoption, operation, and management of any service mesh and their workloads. Meshery's powerful performance management functionality is accomplished through implementation of [Service Mesh Performance](https://smp-spec.io) (SMP). Meshery's cloud native manager functionality leverages [Service Mesh Interface](https://smi-spec.io) (SMI) and Meshery is the conformance tool for SMI. Meshery integrates with Open Application Model (OAM) to enable users to deploy service mesh patterns. Meshery enables operators to deploy WebAssembly filters to Envoy-based data planes. Meshery facilitates learning about functionality and performance of service meshes and incorporates the collection and display of metrics from applications running on or across service meshes.
```
Comment (leecalcote) [inline docs/_adapters/istio/istio.md:64]:
```suggestion
As the open source, cloud native management plane, Meshery enables the adoption, operation, and management of Kubernetes, any service mesh, and their workloads. Meshery's powerful performance management functionality is accomplished through implementation of [Service Mesh Performance](https://smp-spec.io) (SMP). Meshery's cloud native manager functionality leverages [Service Mesh Interface](https://smi-spec.io) (SMI) and Meshery is the conformance tool for SMI. Meshery integrates with Open Application Model (OAM) to enable users to deploy service mesh patterns. Meshery enables operators to deploy WebAssembly filters to Envoy-based data planes. Meshery facilitates learning about functionality and performance of service meshes and incorporates the collection and display of metrics from applications using Prometheus and Grafana integrations.
```

File: docs/pages/concepts/architecture/service-meshes/adapters.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ language: en
 list: include
 ---
 
-As the multi-mesh manager, Meshery offers support for more adapters than any other project or product in the world. Meshery uses adapters for managing the various service meshes.
+As the cloud native manager, Meshery offers support for more adapters than any other project or product in the world. Meshery uses adapters for managing the various service meshes.
```
Comment (leecalcote) [inline docs/pages/concepts/architecture/service-meshes/adapters.md:12]:
```suggestion
As the cloud native manager, Meshery offers support for more service meshes than any other project or product in the world. Meshery uses adapters to offer choice of load generator (for performance management) and for managing different layers of your infrastructure.
```

File: docs/pages/concepts/architecture/service-meshes/index.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ type: service-mesh
 list: exclude
 ---
 
-As the multi-mesh manager, Meshery offers support for more adapters than any other project or product in the world. Learn more about [Meshery Adapters]({{ site.baseurl }}/concepts/architecture/adapters) in the Architecture section.
+As the cloud native manager, Meshery offers support for more adapters than any other project or product in the world. Learn more about [Meshery Adapters]({{ site.baseurl }}/concepts/architecture/adapters) in the Architecture section.
```
Comment (leecalcote) [inline docs/pages/concepts/architecture/service-meshes/index.md:12]:
```suggestion
As the cloud native manager, Meshery offers support for more service meshes than any other project or product in the world. Meshery uses adapters to offer choice of load generator (for performance management) and for managing different layers of your infrastructure.
```

File: docs/pages/es/getting-started/overview.md [hunk 4]
```diff
@@ -29,7 +29,7 @@ Meshery features can be categorized by:
 
 Whether making a Day 0 adoption choice or maintaining a Day 2 deployment, Meshery has useful capabilities in either circumstance. Targeted audience for Meshery project would be any technology operators that leverage service mesh in their ecosystem; this includes developers, devops engineers, decision makers, architects, and organizations that rely on microservices platform.
 
-### Service mesh management - one or multiple service meshes.\*\*
+### Cloud Native manager - one or multiple service meshes.\*\*
```
Comment (leecalcote) [inline docs/pages/es/getting-started/overview.md:32]:
```suggestion
### Cloud Native Manager - multi-cluster Kuberentes and service mesh management.\*\*

```

File: docs/pages/getting-started/overview.md [hunk 5]
```diff
@@ -4,7 +4,7 @@ title: Overview
...
+As the Cloud Native management plane, Meshery enables the adoption, operation, and management of any service mesh and their workloads. Meshery's powerful performance management functionality is accomplished through implementation of [Service Mesh Performance](https://smp-spec.io) (SMP). Meshery's cloud native manager functionality leverages [Service Mesh Interface](https://smi-spec.io) (SMI) and Meshery is the conformance tool for SMI. Meshery integrates with Open Application Model (OAM) to enable users to deploy service mesh patterns. Meshery enables operators to deploy WebAssembly filters to Envoy-based data planes. Meshery facilitates learning about functionality and performance of service meshes and incorporates the collection and display of metrics from applications running on or across service meshes.
```
Comment (leecalcote) [inline docs/pages/getting-started/overview.md:7]:
```suggestion
As the cloud native management plane, Meshery enables the adoption, operation, and management of any service mesh and their workloads. Meshery's powerful performance management functionality is accomplished through implementation of [Service Mesh Performance](https://smp-spec.io) (SMP). Meshery's cloud native management functionality leverages [Service Mesh Interface](https://smi-spec.io) (SMI) and Meshery is the conformance tool for SMI. Meshery integrates with Open Application Model (OAM) to enable users to deploy service mesh patterns. Meshery enables operators to deploy WebAssembly filters to Envoy-based data planes. Meshery facilitates learning about functionality and performance of service meshes and incorporates the collection and display of metrics from applications running on or across service meshes.
```

File: docs/pages/guides/mesheryctl/running-system-checks-using-mesheryctl.md [hunk 6]
```diff
@@ -109,7 +109,7 @@ mesheryctl system check --operator
 **Answer**: _Make sure if you've configured your system to run Meshery in smooth manner. For configuration, do check out the docs site and [this page](https://docs.meshery.io/installation/platforms) to see instructions related to the platform you use._
 
 ##### Question: Do I need a Kubernetes cluster or will a Docker host suffice for Meshery deployments?
-**Answer**: _Meshery's [performance management](functionality/performance-management) functionality does not require a Kubernetes cluster. The rest of Meshery's functionality (e.g. service mesh management) does require a Kubernetes cluster._
+**Answer**: _Meshery's [performance management](functionality/performance-management) functionality does not require a Kubernetes cluster. The rest of Meshery's functionality (e.g. cloud native manager) does require a Kubernetes cluster._
```
Comment (leecalcote) [inline docs/pages/guides/mesheryctl/running-system-checks-using-mesheryctl.md:112]:
```suggestion
**Answer**: _Meshery's [performance management](functionality/performance-management) functionality does not require a Kubernetes cluster. The rest of Meshery's functionality (e.g. cloud native management) does require a Kubernetes cluster._
```

File: docs/pages/guides/mesheryctl/working-with-mesheryctl.md [hunk 7]
```diff
@@ -8,7 +8,7 @@ type: Guides
 category: mesheryctl
 ---
 
-Meshery's command line interface is `mesheryctl`. Use `mesheryctl` to both manage the lifecyle of Meshery itself and to access and invoke any of Meshery's application and service mesh management functions. `mesheryctl` commands can be categorized as follows:
+Meshery's command line interface is `mesheryctl`. Use `mesheryctl` to both manage the lifecyle of Meshery itself and to access and invoke any of Meshery's application and cloud native manager functions. `mesheryctl` commands can be categorized as follows:
```
Comment (leecalcote) [inline docs/pages/guides/mesheryctl/working-with-mesheryctl.md:11]:
```suggestion
Meshery's command line interface is `mesheryctl`. Use `mesheryctl` to both manage the lifecyle of Meshery itself and to access and invoke any of Meshery's application and cloud native management functions. `mesheryctl` commands can be categorized as follows:
```

**Review by leecalcote — CHANGES_REQUESTED** [review]
Use "cloud native management" when replacing "service mesh management"

**Comment by l5io** [comment]
🚀 Preview for commit 4f35da7d4e89d045f52b2ef491cdd075e78d33f9 at: https://62ceaba1b1472205582bc976--meshery-docs-preview.netlify.app

**Review by leecalcote — APPROVED** [review]
Looking real good, @harkiratsm 👍

**Pull request merged**